docs/src/appendix/snforge-library/get_call_trace.md

```diff
@@ -9,104 +9,17 @@ to [`snforge-std` source](https://github.com/foundry-rs/starknet-foundry/tree/v0
 
 Gets current call trace of the test, up to the last call made to a contract.
 
-## Example call trace
```

Comment:
Do we want to move it to the docs?

Reply:
Not really, i don't think it's that useful and it's not really maintainable as well.